dbPedia_17thCentComposers.json

```diff
@@ -0,0 +1 @@
+{"table":[{"name":"dummyname","birthDate":"13-1-1689","birthPlace":"dummyPlace","deathDate":"dummyDeathDate","deathPlace":"dummyDeathPlace ","Pseudonym":"dummyPseudonym ","works":"dummyWorks"},{"name":"Ferdinand III"},{"birthDate":"1608-07-13"},{"birthPlace":"Graz, Austria"},{"deathDate":"1657-04-02"},{"deathPlace":"Graz, Austria"},{"pseudonym":"not given"},{"works":[]},{"name":["Alessandro Scarlatti"]},{"birthDate":"xxxx-xx-xx"},{"birthPlace":"not given"},{"deathDate":""},{"deathPlace":"not given"},{"pseudonym":"not given"},{"works":[]},{"name":["Antonio Vivaldi"]},{"birthDate":"xxxx-xx-xx"},{"birthPlace":"not given"},{"deathDate":""},{"deathPlace":"not given"},{"pseudonym":"not given"},{"works":[]},{"name":["Claudio Monteverdi"]},{"birthDate":"xxxx-xx-xx"},{"birthPlace":"not given"},{"deathDate":""},{"deathPlace":"not given"},{"pseudonym":"not given"},{"works":[]},{"name":["Gregor Aichinger"]},{"birthDate":"xxxx-xx-xx"},{"birthPlace":"not given"},{"deathDate":""},{"deathPlace":"not given"},{"pseudonym":"not given"},{"works":[]},{"name":["Gregorio Allegri"]},{"birthDate":"xxxx-xx-xx"},{"birthPlace":"not given"},{"deathDate":""},{"deathPlace":"not given"},{"pseudonym":"not given"},{"works":[]},{"name":["Henry Purcell"]},{"birthDate":"xxxx-xx-xx"},{"birthPlace":"not given"},{"deathDate":""},{"deathPlace":"not given"},{"pseudonym":"not given"},{"works":["","Love_Is_a_Bourgeois_Construct","","Love_Is_a_Bourgeois_Construct"]},{"name":["Paolo Agostino"]},{"birthDate":"xxxx-xx-xx"},{"birthPlace":"not given"},{"deathDate":""},{"deathPlace":"not given"},{"pseudonym":"not given"},{"works":[]}]}
```

Comment:
This data is just filler data? So I suppose it's not needed?

+ Final scraped data should not make it into the repo